Makefile

```diff
@@ -0,0 +1,11 @@
+execute_dev_checks: check_security test check_code_style
+
+check_security:
+	symfony check:security
+
+test:
+	bin/phpunit
+
+check_code_style:
+	php vendor/bin/phpcs --standard=psr2 src/ -n
+	php vendor/bin/phpcs --standard=psr2 tests/ -n
```

Comment:
@daria-sieroshtan 
1. Are you sure you want this standard?
1. People usually don't want to check code style in tests, some rules (like maximum line length) are really hard to comply with and this is not a production code anyway